doc/newfeat.texi

```diff
@@ -4,6 +4,8 @@
 Changes and New Features in 19.04 (unreleased):
 @itemize @bullet
 
+@item ESS[R]: Improved @code{xref} lookup (@kbd{M-.})
```

Comment:
This needs an explanation of what was improved.

> I wonder how we can popularize ess-r-package-library-paths a bit more. I expect few people know that in order to make xref work that one is necessary.

Maybe mention it here? I don't use this variable personally.